features/lib/support/env.rb

```diff
@@ -22,9 +22,16 @@
 end
 
 Around do |_, block|
+  original_publish_token = ENV['CUCUMBER_PUBLISH_TOKEN']
+  ENV.delete('CUCUMBER_PUBLISH_TOKEN')
```

Comment:
The value deleted from the hash is returned, so you can assign it:

```ruby
  original_publish_token = ENV.delete('CUCUMBER_PUBLISH_TOKEN')
```

Reply:
done ✅